### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
- Creation of the variable folder, and the variables that will be used in the project were configured.
- Refactoring the index.html with the first title of the page.
- Page Titles card completed

## Screenshots
image

## Trello Card
https://trello.com/c/tEg5JwZ8

### CODE REVIEW

File: maquetado/scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.main-title{
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: leave a space between `.main-title` and `{`

File: maquetado/index.html [hunk 2]
```diff
@@ -8,6 +8,13 @@
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
   <body>
-    <h1 class="title">Hello World</h1>
+      <main class="main-project">
+          <section class="section-project">
+              <div class="main-title-container">
+                  <img src="../assets/triangle.svg" alt="triangle-wolox" class="main-title-img">
+                  <h1 class="main-title">Front-End Tools</h1>
+              </div>
```
Comment (CarlosHWolox) [inline maquetado/index.html]: Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the `h1` element and apply the image as a decorator using only CSS (hint: you can use the `::before` selector), that way you will have a single reusable html tag (the `h1`) that contains the text, and manage the decorative part of the text via CSS only.  If you have any doubt about this ask me.

File: maquetado/scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  font-size: $text-big;
+  left: -20px;
+  position: relative;
+  top: 5px;
+
+  &-img{
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: Same here

File: maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    height: 100%;
+    max-height: $title-decoration-height;
+    max-width: $title-decoration-width;
+    width: 100%;
+  }
+
+  &-container {
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: Same here.

File: maquetado/scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.main-title {
+  align-items: center;
+  display:  flex;
+  font-size: $text-big;
+  
+  &:before {
+    background: url('../assets/triangle.svg')  no-repeat;
+    content: '';
+    display:inline-block;
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: Missing a space here. And why it is necessary this?

File: maquetado/scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    height:  $title-decoration-height;
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: Doble space here haha

File: maquetado/scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    left: 20px;
+    max-height: 100%;
+    max-width: 100%;
```
Comment (CarlosHWolox) [inline maquetado/scss/titles.scss]: why you need the `max-width` and `max-height`?

Comment (DavidRojas1612) [reply maquetado/scss/titles.scss]: I'm sorry, I confused a part of the theory of the style guide.

File: maquetado/scss/titles.scss (hunk 5 above)
Comment (DavidRojas1612) [reply maquetado/scss/titles.scss]: it's really not necessary.

**Pull request merged**